plugins/woocommerce/client/admin/client/settings-payments/components/ellipsis-menu-content/ellipsis-menu-content.tsx

```diff
@@ -81,6 +116,9 @@ export const EllipsisMenuContent = ( {
 			} );
 	};
 
+	/**
+	 * Disables the payment gateway.
```

Comment:
"Disables the payment gateway from payment processing."

Reply:
thanks! addressed in https://github.com/woocommerce/woocommerce/pull/54263/commits/1cb4f8eb9290355d38f8cefebb9784d70bdf6450